x-pack/test/api_integration/apis/management/index_management/component_templates.ts

```diff
@@ -118,7 +121,7 @@ export default function ({ getService }: FtrProviderContext) {
       after(async () => {
         // Clean up any component templates created in test cases
         await Promise.all(
-          [COMPONENT_NAME, REQUIRED_FIELDS_COMPONENT_NAME].map(deleteComponentTemplate)
+          [COMPONENT_NAME, REQUIRED_FIELDS_COMPONENT_NAME].map(removeComponentTemplate)
         ).catch((err) => {
           // eslint-disable-next-line no-console
           console.log(`[Cleanup error] Error deleting component templates: ${err.message}`);
```

Comment:
nit: there is a logger service, that could be used instead of console `const log = getService('log');`